UK_Reports/reports/external_layouts.xml

```diff
@@ -102,17 +99,24 @@
 <!-- New external layout for UK headers/footers which is callable per report -->
 <template id="external_layout_uk">
 	<!-- Multicompany -->
+	<t t-set="company" t-value="False"/>
```

Comment:
Probably best to take a look at core 13.0 `external_layout` as i presume there is no concept of `t set="company"` there. Also setting it to false at the top and then having a bunch of checks for `t-if="not company"` doesn't make sense as it's always not there, so i think a lot of stuff can be removed here (i presume you've done this as there's no multicompany in 13?)

Reply:
I made a change, to make it little simpler the code. In fact when i was testing one of the reports the model didnt have company_id field and was giving error. I thought not to change a lot functionality of existing reports(Considering the multicompany scenario) and i set the value of company from user if not found in record